bigtable-client-core-parent/bigtable-client-core/pom.xml

```diff
@@ -118,6 +194,14 @@ limitations under the License.
                     <groupId>com.google.guava</groupId>
                     <artifactId>guava-jdk5</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>com.google.http-client</groupId>
+                    <artifactId>google-http-client</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>com.google.http-client</groupId>
+                    <artifactId>google-http-client-jackson2</artifactId>
+                </exclusion>
```

Comment:
How these deps need to pulled out/up? Does anything other than google-auth-library-oauth2-http use http-client? if not, it might better to remove the exclusions and the toplevel deps